python/whylogs/core/datatypes.py

```diff
@@ -107,6 +107,21 @@ def _do_match(cls, dtype_or_type: Any, maybe_type: Optional[Any]) -> bool:
         return False
 
 
+class Tensor(DataType[np.ndarray]):
+    def __init__(self) -> None:
+        super().__init__(np.ndarray)
+
+    @classmethod
+    def _do_match(cls, dtype_or_type: Any, maybe_type: Optional[Any]) -> bool:
+        if maybe_type:
+            dtype_or_type = maybe_type
+
+        if not isinstance(dtype_or_type, type):
+            return False
+
+        return dtype_or_type == np.ndarray  # TODO: can't see the data type in the ndarray :(
```

Comment:
This is bad :(  Schema matching by `Tensor` shouldn't match `ndarray` of strings.

Reply:
Recommend only resolving tensor columns by name, or avoid having non-tensorable columns of `ndarray`.

Reply:
Ditching `Tensor`, resolve by name for now...